resources/ts/components/ImageField/ImageField.tsx

```diff
@@ -2,125 +2,136 @@ import React, { useState } from 'react';
 import styled, { css } from 'styled-components';
 
 import { isBlobURL } from '@wordpress/blob';
-import { MediaPlaceholder, MediaReplaceFlow, MediaUploadCheck } from '@wordpress/block-editor';
+import {
+	MediaPlaceholder,
+	MediaReplaceFlow,
+	MediaUploadCheck,
+} from '@wordpress/block-editor';
 import { Button, Flex, Spinner } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 
 import { useEditDegreeProgram } from '../../contexts/DegreeProgramEditFormProvider';
 import { useFieldContext } from '../FormField/FormField';
 import useMedia from './useMedia';
 
-import { Image } from 'defs';
+import { Image } from '../../defs';
 
 type ImageFieldProps = {
-    path: 'teaser_image' | 'featured_image';
-    title?: string;
+	path: 'teaser_image' | 'featured_image';
+	title?: string;
 };
 
 const THUMBNAIL_SIZE = 150;
 
-const StyledWrapper = styled.div<{ hasImage: boolean }>`
+const StyledWrapper = styled.div< { hasImage: boolean } >`
     .components-placeholder__fieldset {
-        ${({ hasImage }) =>
-            hasImage
-                ? css`
-                      .components-form-file-upload,
-                      .block-editor-media-placeholder__cancel-button,
-                      .components-button.is-tertiary {
-                          display: none;
-                      }
-                  `
-                : ''}
+        ${ ( { hasImage } ) =>
+			hasImage
+				? css`
+						.components-form-file-upload,
+						.block-editor-media-placeholder__cancel-button,
+						.components-button.is-tertiary {
+							display: none;
+						}
+				  `
+				: '' }
         };
     }
 `;
```

Comment:
We have a mix of spaces and tabs here. Any chance we can align on tabs? Thanks!
Edit: The automated white space change seems to have problems with styled-components, see, e.g., https://github.com/RRZE-Webteam/FAU-Studium/pull/126/files#diff-cd8e95a5defba7a04b47061203039f3810f3c770f2f984f55c75c10904eb2dce; could you please check all these components? Thanks!